ethstaker_deposit/cli/generate_keys.py

```diff
@@ -163,21 +159,49 @@ def generate_keys_arguments_decorator(function: Callable[..., Any]) -> Callable[
     return function
 
 
+def get_amount_prompt_from_template() -> str:
+    ctx = click.get_current_context(silent=True)
+    chain = ctx.params.get('chain', 'mainnet') if ctx is not None else 'mainnet'
+    devnet_chain_setting = ctx.params.get('devnet_chain_setting')
+    if devnet_chain_setting is not None:
+        chain_setting = devnet_chain_setting
+    else:
+        chain_setting = get_chain_setting(chain)
+    min_deposit = chain_setting.MIN_DEPOSIT_AMOUNT
+    activation_amount = chain_setting.MIN_ACTIVATION_AMOUNT
+    template = load_text(['arg_amount', 'prompt'], func='generate_keys_arguments_decorator')
+    return template.format(min_deposit=min_deposit, activation_amount=activation_amount)
+
+
+def get_default_amount() -> str:
+    ctx = click.get_current_context(silent=True)
+    chain = ctx.params.get('chain', 'mainnet') if ctx is not None else 'mainnet'
+    devnet_chain_setting = ctx.params.get('devnet_chain_setting')
```

Comment:
Use None as the default if `'devnet_chain_setting'` is not found as a key in this dict.